### ISSUE TITLE
feature: Player toggle in lobby

### ISSUE THREAD

**Issue by john-bv** [Zeqa-Network/Issue-Tracking#8]
## Key Concepts

Originally posted by **@bourah**, [view suggestion thread here](https://discord.com/channels/874571610126942208/1125751051371417690/1125751051371417690)
> a button to turn players invisible in the hub as in low end devices it lags like its born to lag

**Comment by Silentmussle913** [comment]
yep im the orignal poster btw in other words bourah/Silentmussle913

**Comment by Silentmussle913** [comment]
if anyone is trying to find a server with this feature fully working try Venity its asia so 90% cant play yet this feature is god damn fire!. after zeqa impliments this this feature will we hella helpful for people with low end devices. Ip for Venity is (play.venitymc.com) i am not affilated with venity in any ways and i dotn even play often i just like the feature and the ip is not self promo stuff its a genuinely for people who wants to see the fps difference
